includes/class-comment.php

```diff
@@ -111,6 +113,11 @@ public static function are_comments_allowed( $comment ) {
 			return false;
 		}
 
+		if ( is_single_user() && \user_can( $current_user, 'activitypub' ) ) {
```

Comment:
```suggestion
		if ( is_single_user() && \user_can( $current_user, 'publish_posts' ) ) {
```

Maybe we should keep checking `publish_posts`? If someone uses the blog-only mode, he might not care about setting up the caps properly?!?